### ISSUE TITLE
Single GPU?

### ISSUE THREAD

**Issue by Yuliang-Zou** [YudeWang/deeplabv3plus-pytorch#15]
Hi, I wonder why only multi-GPU is allowed currently. If I replace the synchronized BN with normal BN and use PyTorch built-in DataParallel, can I train it with only 1 GPU? Also, is single-GPU inference supported at this moment?

**Comment by YudeWang** [comment]
@Yuliang-Zou It is fine to run on single gpu but there are some code should be rewritten. Not only synchronized BN, there are lots of '.to(1)' operations (move tensors to same gpu for loss calculation) should be removed. And the batchesize should be changed according to gpu memory.
